packages/bundler/src/interfaces/IBundler.ts

```diff
@@ -1,9 +1,9 @@
-import { UserOpResponse, UserOpGasResponse, UserOpReceipt, UserOpByHashResponse } from "../utils/Types";
+import { UserOpResponse, UserOpGasResponse, UserOpReceipt, UserOpByHashResponse, SendUserOpOptions } from "../utils/Types";
 import { UserOperation } from "@biconomy/core-types";
 
 export interface IBundler {
   estimateUserOpGas(_userOp: Partial<UserOperation>): Promise<UserOpGasResponse>;
-  sendUserOp(_userOp: UserOperation): Promise<UserOpResponse>;
+  sendUserOp(_userOp: UserOperation, _params?: SendUserOpOptions): Promise<UserOpResponse>;
```

Comment:
some places it is params some places it is _params. keep one

also try not to add anything in interfaces and abstract contracts (if their type is not used directly. like SA we use exact class type, for bundler also if its Bundler and ot IBundler can only have this in child and not interfaces)

Reply:
the issue is the account package variable `bundler` type is not the Bundler, it's `IBundler` so if the method is not mentioned there it will throw errror. Or I have to update the account package from 
<img width="537" alt="image" src="https://github.com/bcnmy/biconomy-client-sdk/assets/42104907/a6d13d5b-979e-48ae-bda0-8451a6b695ec">

Regarding the `_params`. Have to update all the unused variables with _ before to avoid lint warning, majorly in interface for all variables.

Reply:
ok, bundler is fine but in BaseSmartAccount it can be avoided to add optional params and just add in the SA classes. 

and _params case try to keep it consistent then, at least it must be same way in particular file.

Reply:
> and _params case try to keep it consistent then, at least it must be same way in particular file.

updated

> ok, bundler is fine but in BaseSmartAccount it can be avoided to add optional params and just add in the SA classes.

I don't think this is possible because the SA class is not calling the bundler at all, the BaseSmartAccount class calls the bundler and it should have the context to that file.

Reply:
> and _params case try to keep it consistent then, at least it must be same way in particular file.

updated

> ok, bundler is fine but in BaseSmartAccount it can be avoided to add optional params and just add in the SA classes.

This will work for the SA class but I don't think this is possible because the SAV2 class is not calling the bundler, the BaseSmartAccount class calls the bundler and it should have the context to that file.